libcloudforensics/gcp.py

```diff
@@ -658,6 +681,83 @@ def ExecuteFunction(self, function_name, args):
     return function_return
 
 
+class GoogleCloudBuild(GoogleCloudProject):
+  """Class to call Google Cloud Build APIs.
+
+  Attributes:
+    gcb_api_client: Client to interact with GCB APIs.
+  """
+  CLOUD_BUILD_API_VERSION = 'v1'
+
+  def __init__(self, project_id):
+    """Initialize the GoogleCloudBuild object.
+
+    Args:
+      project_id (str): The name of the project.
+    """
+
+    self.gcb_api_client = None
+    super(GoogleCloudBuild, self).__init__(project_id)
+
+  def GcbApi(self):
+    """Get a Google Cloud Build service object.
+
+    Returns:
+      apiclient.discovery.Resource: A Google Cloud Build service object.
+    """
+
+    if self.gcb_api_client:
+      return self.gcb_api_client
+    self.gcb_api_client = CreateService(
+        'cloudbuild', self.CLOUD_BUILD_API_VERSION)
+    return self.gcb_api_client
+
+  def CreateBuild(self, build_body):
+    """Create a cloud build.
+    Args:
+      build_body (dict): the build resource describes how to find the source
+        code and how to build it.
+    Returns:
+      dict: represents long-running operation that is the result of
+        a network API call.
+    """
+    cloud_build_client = self.GcbApi().projects().builds()
+    build_info = cloud_build_client.create(
+        projectId=self.project_id, body=build_body).execute()
+    build_metadata = build_info['metadata']['build']
+    log.info(
+        'Build started, logs bucket: {0:s}, logs URL: {1:s}'.format(
+            build_metadata["logsBucket"], build_metadata["logUrl"]))
+    return build_info
+
+  def BlockOperation(self, response):  #  # pylint: disable=arguments-differ
+    """Block execution until API operation is finished.
+
+    Args:
+      response (dict): Google Cloud Build API response.
+
+    Returns:
+      str: Operation result in JSON format.
+
+    Raises:
+      RuntimeError: If API call failed.
+    """
+    service = self.GcbApi()
+    while True:
+      request = service.operations().get(name=response['name'])
+      response = request.execute()
+      if response.get('done', None) and response.get('error', None):
```

Comment:
Since `dict.get()` returns `None` by default when a key is not found, this might be more readable to shorten these statements to just `response.get('done')`

Reply:
done.